misc/python/materialize/output_consistency/common/configuration.py

```diff
@@ -40,6 +41,7 @@ def __init__(
         self.max_iterations = max_iterations
         self.avoid_expressions_expecting_db_error = avoid_expressions_expecting_db_error
         self.skip_postgres_incompatible_types = skip_postgres_incompatible_types
+        self.generate_nested_expressions = generate_nested_expressions
```

Comment:
Probably not. I added that at the beginning when I was not yet sure how well it would work.

Reply:
I think we need both plain and nested expressions . The more nested an expression is, the more likely it is going to return NULL , or `FALSE`, or be quickly determined as being Constant, even if not over a constant view, or otherwise not exercise a lot of code paths. Complex expressions in the `WHERE` clause usually lead to no rows being returned at all.

Reply:
This is true, @philip-stoev, but this option does not prevent generating non-nested expressions. It just enables generating **also** nested expressions.

Therefore, I consider removing this option.

Reply:
✅ Removed.